source/ios/tools/IOSFeed/IOSFeed.cs

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Blob;
+
+namespace IOSFeedNS
+{
+    public static class Constatns
+    {
+        public const string connectionStringPath = "source/ios/tools/IOSFeed/ConnectString.txt";
+        public const string containerId = "adaptivecardsiosblobs";
+        public const string frameworkPath = "source/ios/AdaptiveCards/AdaptiveCards";
+        public const string frameworkName = "AdaptiveCards.framework.zip";
+        public const string podspecPath = "./source/ios/tools/";
+        public const string targetPodspecPath = "./source/ios/";
+        public const string podspecName = "AdaptiveCards.podspec";
+    }
+    class IOSFeed
```

Comment:
Nit: a quick class summary/description would be handy here) ; is there a readme.md or something we can add as part of this btw? Thanks!